resources/web/docs_js/components/console_widget.jsx

```diff
@@ -33,23 +33,25 @@ export class _ConsoleForm extends Component {
 
     return (
       <form>
+         <p><strong>Want to use the Kibana Console?</strong></p>
```

Comment:
Let's add a title to this window:

`Kibana settings`

I like how Cloud separates the trial settings:
![Screenshot 2024-04-11 at 12 57 46 PM](https://github.com/elastic/docs/assets/38990715/aac2d827-267b-4490-a5c6-88b9175192f8)

Secondary header: `Cloud deployment`
`Console URL` 
Below the field: In Kibana, open the main menu, go to Dev Tools > Console, then copy the URL.